examples/wasm/wasm_sample/Makefile

```
@@ -104,6 +104,9 @@ COMPILE_FLAGS := -Wall \
 %.wat: %.wasm Makefile
 	wasm2wat -o $@ $<
 
+.PHONY: all
+all: hello.wasm
```

I would expect al to build everything ->

```suggestion
.PHONY: default
default : hello.wasm
```

or 

```suggestion
hello.wasm:
```